sycl/include/CL/sycl/accessor.hpp

```diff
@@ -381,39 +391,49 @@ class image_accessor
         T, detail::type_list<cl_int4, cl_float4>>::type::value;
   };
 
-  //  Available only when: (accessTarget == access::target::image ||
-  //  accessTarget == access::target::host_image) && accessMode ==
-  //  access::mode::read
+  // Available only when:
+  // (accessTarget==access::target::image && accessMode==access::mode::read) ||
```

Comment:
Please, add space around binary operators.
```suggestion
  // (accessTarget == access::target::image && accessMode == access::mode::read) ||
```

Reply:
done